files/zh-cn/learn/javascript/first_steps/silly_story_generator/index.md

```diff
@@ -172,8 +172,8 @@ slug: Learn/JavaScript/First_steps/Silly_story_generator
 4. 在第一个 `if` 块中再次调用这个字符串替换方法，以使 `newStory` 字符串中的名字“李雷”替换为变量 `name` 的值。这里我们说：“如果 `customName` 中有值，就把故事里的“李雷”替换成它。”如果是汉化版将 newStory 中的“李雷”替换成 `name` 的值；
 5. 在第二个 `if` 块中检查 `american` 单选按钮是否被选中。如果选中，就要将故事中的重量和温度值从公斤和摄氏度转换为磅和华氏度，具体事项如下：
 
-   1. 确定英美单位的转换公式。
-   2. 定义变量 `weight`、`temperature` 的行中，分别将美制单位按公式转化为英制，用 `Math.round()` 对计算结果取整。然后将英式单位连接到末尾。
+   1. 确定公制和美制单位的转换公式。
```

Comment:
Please refer the format of [the en-US document](https://github.com/mdn/content/blob/71251662105fbac882aca295c641500175bb9ef1/files/en-us/learn/javascript/first_steps/silly_story_generator/index.md?plain=1#L91)